src/components/modal.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./modal.css";
+import PropTypes from "prop-types";
```

Comment:
I said this before, but this is not the PR to introduce `PropTypes`. We can introduce it in another PR.

Note that because we run the devserver in production, this will actually slow down production if we include it without thinking.

Reply:
We also don't need it in `package.json` in this PR.